spec/plurimath/latex_spec.rb

```diff
@@ -24,9 +24,8 @@
       let(:string) { '\\cos{45}' }
       it 'returns parsed Latex to Formula' do
         expected_value = Plurimath::Math::Formula.new([
-          Plurimath::Math::Function::Cos.new(
-            Plurimath::Math::Number.new("45")
-          )
+          Plurimath::Math::Symbol.new("cos"),
+          Plurimath::Math::Number.new("45"),
```

Comment:
I'm concerned about this change. The previous structure seems more correct to me, and is likely necessary when we do Content MathML (Semantic MathML, not Presentation MathML).

Reply:
@opoudjis says he did not ask for this change, so let's keep the previous expectation.

Reply:
From what I can tell:

* The source in my equation did not use curly brackets: \sin \lambda_0
* Curly brackets are required if the argument is more than one character long: \sin 2x should be \sin{2x}. (That's more obvious for something like y^{2x}
* However, \lambda_0 counts as a single object, so the curly brackets are not needed to make it a single argument of sin
* The LatexMath we have been using until now was processing that with sin as a `<mi>` instead of a `<mo>`. I did not notice that, and that is clearly a bug on their part, which we should not be replicating. Sorry for not spotting it. (That's why I did not list it as an issue to fix.)

* \cos 5 is processed as cos(5), but by that rule of single character arguments, \cos 45 is processed as cos(4) 5
* So properly speaking, \cos 45 is not right, and you are entitled to parse it somehow else
* ... But I still wouldn't

Reply:
> ... But I still wouldn't

`\cos 45` ought to be parsed as `cos(45)`. Clearly the string token includes "45". So `cos(45)` is right.